static_src/stores/base_store.js

```diff
@@ -9,6 +9,10 @@ import Immutable from 'immutable';
 import AppDispatcher from '../dispatcher.js';
 import LoadingStatus from '../util/loading_status.js';
 
+// TODO clean up the listeners and reduce the MAX_LISTENERS to 15
+const MAX_LISTENERS = 20;
+const MAX_LISTENERS_THRESHOLD = 10;
```

Comment:
Why should 10 me the threshold? It seems like the max is there to catch possible leaks, but we're knowingly using this man, so I'd expect to not be warned for it. Or is there other problems with having so many listeners?